docker/device_startup.docker

```diff
@@ -64,20 +60,21 @@ RUN python setup.py develop
 #==============================================================================
 #Install python deps
 
-RUN conda install astropy
-RUN conda install matplotlib
-RUN conda install requests
-RUN conda install scipy
+RUN conda install astropy matplotlib requests scipy
+RUN pip install pymongo netifaces pyro4 pyyaml
+
+#==============================================================================
+#Setup permissions for $PANUSER; specify $PANUSER as default user
 
-RUN pip install pymongo
-RUN pip install netifaces
-RUN pip install pyro4
-RUN pip install pyyaml
+RUN chmod 1777 /tmp 
+RUN chown -R $PANUSER $PANDIR && mkdir /home/$PANUSER \
+    && chown -R $PANUSER /home/$PANUSER
+USER $PANUSER
 
 #==============================================================================
 
 #For some reason this is necessary
 RUN rm -r $PANDIR/images
```

Comment:
you just created this... is this because the soft link(?) to `sshfs` mount point can't be created later on?

Reply:
Uh, yeah, what's the story here?